python/delta/tables.py

```diff
@@ -594,6 +594,27 @@ def addFeatureSupport(self, featureName: str) -> None:
         DeltaTable._verify_type_str(featureName, "featureName")
         self._jdt.addFeatureSupport(featureName)
 
+    @since(4.0)  # type: ignore[arg-type]
+    def dropFeatureSupport(self, featureName: str) -> None:
+        """
+        Modify the protocol to drop a supported feature. The operation always normalizes the
+        resulting protocol. Protocol normalization is the process of converting a table features
+        protocol to the weakest possible form. This primarily refers to converting a table features
+        protocol to a legacy protocol. A Table Features protocol can be represented with the legacy
+        representation only when the features set of the former exactly matches a legacy protocol.
+        Normalization can also decrease the reader version of a table features protocol when it is
+        higher than necessary.
+
+        Normalization can also decrease the reader version of a table features protocol when it is
```

Comment:
Is the repeat of " Normalization can also decrease the reader version of a table features protocol when it is
higher than necessary." intended?